streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java

```diff
@@ -280,17 +284,8 @@ public void shouldLogAndMeterWhenSkippingExpiredWindow() {
             driver.pipeInput(recordFactory.create(topic, "k", "6", 6L));
             LogCaptureAppender.unregister(appender);
 
-            final MetricName metricName = new MetricName(
-                "late-record-drop-total",
-                "stream-processor-node-metrics",
-                "The total number of occurrence of late-record-drop operations.",
-                mkMap(
-                    mkEntry("client-id", "topology-test-driver-virtual-thread"),
-                    mkEntry("task-id", "0_0"),
-                    mkEntry("processor-node-id", "KSTREAM-AGGREGATE-0000000001")
-                )
-            );
-            assertThat(driver.metrics().get(metricName).metricValue(), equalTo(7.0));
```

Comment:
This class has two different tests that check the metrics, so I pulled it out into a function.